mindsdb/api/http/namespaces/databases.py

```diff
@@ -97,3 +103,140 @@ def delete(self, database_name):
                 + f'Full error: {e}. '
                 + 'Please check the name and try again.')
         return '', HTTPStatus.NO_CONTENT
+
+
+def _tables_row_to_obj(table_row: TablesRow) -> Dict:
+    type = table_row.TABLE_TYPE.lower()
+    if table_row.TABLE_TYPE == 'BASE TABLE':
+        type = 'data'
+    return {
+        'name': table_row.TABLE_NAME,
+        'type': type
+    }
+
+
+@ns_conf.route('/<database_name>/tables')
+class TablesList(Resource):
+    @ns_conf.doc('list_tables')
+    def get(self, database_name):
+        '''Get all tables in a database'''
+        session = SessionController()
+        datanode = session.datahub.get(database_name)
+        all_tables = datanode.get_tables()
+        table_objs = [_tables_row_to_obj(t) for t in all_tables]
+        return table_objs
+
+    @ns_conf.doc('create_table')
+    def post(self, database_name):
+        '''Creates a table in a database'''
+        if 'table' not in request.json:
+            abort(HTTPStatus.BAD_REQUEST, 'Must provide "table" parameter in POST body')
```

Comment:
I think we need to have single way to format HTTP errors. `abort` returns to client json with
```
{message: 'Must provide "table" parameter in POST body'}
```
But in some other places we return error in [this form](https://github.com/mindsdb/mindsdb/blob/staging/mindsdb/api/http/utils.py#L7). I prefer second form, because it based on RFC proposals. It is not standard, but at least some kind of agreement. What do you think? If you want, please keep `abort` and merge PR